controllers/reconcile.go

```diff
@@ -112,6 +112,14 @@ func populateContainerVolumeMounts(frontendEnvironment *crd.FrontendEnvironment)
 		MountPath: "/opt/app-root/src/build/stable/operator-generated",
 	})
 
+	if frontend.Name != "chrome" {
```

Comment:
We don't plan on leaving this in, right? We're just cutting some slack to coordinate with chrome to change their dist dir layout?

Reply:
@adamrdrew correct; we will remove in another PR.